### ISSUE TITLE
Val loss increases after 13 epochs

### ISSUE THREAD

**Issue by harshays572** [pythonlessons/TensorFlow-2.x-YOLOv3#196]
val loss increases after 13 epochs

**Comment by pythonlessons** [comment]
Why is this an issue? Maybe the model is overfitting with your data?

**Comment by harshays572** [comment]
even i though the same so i tried with 3 classes but then i got Train and Val loss as NaN.

**Comment by harshays572** [comment]
i used init and end learning rate 1e-4, 1e-6 respectively.

**Comment by harshays572** [comment]
The problem is in Calculating Location Loss Because when i print them while model Training, Location Loss gives NaN.
